packages/next-codemod/lib/run-jscodeshift.ts

```diff
@@ -9,7 +9,20 @@ export default function runJscodeshift(
   // we run jscodeshift in the same process to be able to
   // share state between the main CRA transform and sub-transforms
   return Runner.run(transformerPath, files, {
-    ignorePattern: ['**/node_modules/**', '**/.next/**', '**/build/**'],
+    ignorePattern: [
+      '**/node_modules/**',
+      '**/.next/**',
+      '**/build/**',
```

Comment:
This is too generic. We should include it and leave it to user space to configure.